### PR TITLE
store num of sending feedback

### PR DESCRIPTION
**Pull request by moatazalaa14**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 
-REACT_APP_POSTS_SERVICE_URL='http://localhost:3001/'
+REACT_APP_POSTS_SERVICE_URL='https://pickly-posts-be-dev.m3ntorship.net/'
```
Comment (Nilegfx) [inline .env:3]: don't change this file

**Review by Nilegfx — CHANGES_REQUESTED** [review]
why would you store the number of feedback, this is already handled by the backend.

having two different places to configure this is tricky and over-engineering.

**Comment by moatazalaa14** [comment]
> why would you store the number of feedback, this is already handled by the backend.
> 
> having two different places to configure this is tricky and over-engineering.

**Comment by moatazalaa14** [comment]
> why would you store the number of feedback, this is already handled by the backend.
> 
> having two different places to configure this is tricky and over-engineering.

@Nilegfx 
It's for any user send more than 5 feedback it make button disabled and make it gray for 1 day to be able to send a new request . i think that number of sending feedback for one user it is not handled from back 

i dont understand what do you mean two places localstorage and what ?😅

**Review by Nilegfx — CHANGES_REQUESTED** [review]
> It's for any user send more than 5 feedback it make button disabled and make it gray for 1 day to be able to send a new request . i think that number of sending feedback for one user it is not handled from back

is this PR ready for review, or you still working on it?


> i dont understand what do you mean two places localstorage and what ?😅

localStorage and the backend (both do the validation)

**Pull request closed without merging**